cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2004,3 +2004,74 @@ action.summary_index = 1
 action.summary_index._name = cyences
 action.summary_index.category = Authentication
 action.summary_index.alert_name = Authentication - Excessive Failed VPN Logins from a Source
+
+
+
+# ===============
+# Linux/Unix
+# ===============
+[Linux - Local User List - Lookup Gen]
+disabled = 1
+enableSched = 1
+alert.track = 0
+cron_schedule = 53 * * * *
+description = This report generate lookup of linux local accounts.\
+\
+Data Collection : Below two scripted inputs must be enable. \
+                  Splunk_TA_nix Add-on -> usersWithLoginPrivs.sh \
+                  Cyences add-on for Splunk -> sudousers.sh
+dispatch.earliest_time = -62m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_linux_users_with_previledge`\
+| table _time host UID USERNAME GID HOME_DIR \
+| eval user_discovery=_time \
+| stats earliest(_time) as _time latest(*) as * by host UID \
+| append \
+    [| search  `cs_linux_sudousers`\
+    | stats latest(_raw) as raw latest(_time) as sudo_discover by host \
+    | rex field=raw "sudouser=(?<USERNAME>.*)" max_match=0 \
+    | fields - raw \
+    | mvexpand USERNAME \
+    | eval sudo_access="Yes" ] \
+| stats first(sudo*) as sudo*_new first(user_discovery) as user_discovery_new first(*) as * by host, USERNAME \
+| appendpipe \
```

Comment:
We don't need appendpipe command of subsearch.

Reply:
bro, I specifically want to add results from lookup after we get whole results from above search. I am not sure how append in "| inputlookup" works.

Reply:
For that we can use | inputlookup append=True directly.